src/components/header/Header.css

```diff
@@ -1,3 +1,84 @@
 * {
   box-sizing: border-box;
 }
+
+.Bookstore-CMS {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 1.038rem 0.125rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+a {
+  color: #121212;
+}
+
+a:active {
+  color: #121212;
+}
+
+.bookstore-header {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+  height: 5.938rem;
+  margin: 0;
+  padding: 1.438rem 9.088rem 1.688rem 10.75rem;
+  background-color: #fff;
+}
```

Comment:
- [ ] As pointed out earlier, can you please ensure that the contents of the header (`Bookstore-cms` and the `logo`) is well aligned with the other elements according to the given template. 👍 Kindly don't be discouraged, reproducing perfectly a given design shows how well we pay attention to details as developers; and I'm sure recruiters are always on look out for such. ℹ️ 

<img width="941" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/230744431-85b3fcf4-4089-4b1d-a008-d6e7ee92e57e.png">

Reply:
Hey @OyePriscilla, Thank you again for the review. I make changes according to the requested change. Thanks